components/test-suite/.ant-targets-build.xml

```diff
@@ -0,0 +1,64 @@
+bundle
```

Comment:
What's this file used for?  It's not in the original PR, and it doesn't appear to be used in this one either.

Reply:
Unexpected addition while fixing conflicts on my commit cherry-pick. Thanks for spotting promptly.